PowerEditor/src/MISC/PluginsManager/Notepad_plus_msgs.h

```diff
@@ -1017,13 +1017,10 @@ enum Platform { PF_UNKNOWN, PF_X86, PF_X64, PF_IA64, PF_ARM64 };
 	//  	...
 	//  }
 
-	#define NPPM_GETTOOLBARICONSETMODE (NPPMSG + 118)
-	// BOOL NPPM_GETTOOLBARICONSETMODE(0, 0)
-	// Get Notepad++ Toobar Icon Set mode (ENUM Range: 0 to 4).
-	// wParam: 0 (not used)
-	// lParam: 0 (not used)
-	// Return Toolbar Icon Set mode as an integer value [ENUM range: 0 (TB_SMALL - Fluent UI: small) to 4 (TB_STANDARD - Standard icons: small)].
-
+	#define NPPM_GETTOOLBARICONSETCHOICE (NPPMSG + 118)
+	// BOOL NPPM_GETTOOLBARICONSETCHOICE(0, 0)
+	// Return toolbar icon set choice as an integer value. Here are 5 possible values:
+	// 0 (Fluent UI: small), 1 (Fluent UI: large), 2 (Filled Fluent UI: small), 3 (Filled Fluent UI: large) and 4 (Standard icons: small).

```

Comment:
Please use the following comments:
```cpp
	#define NPPM_GETTOOLBARICONSETCHOICE (NPPMSG + 118)
	// BOOL NPPM_GETTOOLBARICONSETCHOICE(0, 0)
	// Get Notepad++ toobar icon set choice (Fluent UI: small, Fluent UI: large, Filled Fluent UI: small, Filled Fluent UI: large and Standard icons: small.
	// wParam: 0 (not used)
	// lParam: 0 (not used)
	// Return toolbar icon set choice as an integer value. Here are 5 possible values:
	// 0 (Fluent UI: small), 1 (Fluent UI: large), 2 (Filled Fluent UI: small), 3 (Filled Fluent UI: large) and 4 (Standard icons: small).
```

Reply:
Done. Modified the comment line. In the 4th commit.